src/StrokerCache/Strategy/CacheAllExcept.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace StrokerCache\Strategy;
+
+use Zend\Mvc\MvcEvent;
+
+use StrokerCache\Strategy\AbstractStrategy as AbstractStrategyStrategyStrokerCache;
+use StrokerCache\Exception\BadConfigurationException;
+
+class CacheAllExcept extends AbstractStrategyStrategyStrokerCache
+{
+    protected $except;
+
+    public function shouldCache(MvcEvent $event)
```

Comment:
Could you add the missing docblocks to the methods in this class? This one can be {@inheritDoc} because it is already in the interface definition